library/psa_crypto_driver_wrappers.h

```diff
@@ -328,6 +328,22 @@ psa_status_t psa_driver_wrapper_mac_verify_finish(
 psa_status_t psa_driver_wrapper_mac_abort(
     psa_mac_operation_t *operation );
 
+
+/*
+ * Entropy entry points
+ */
+#define PSA_DRIVER_GET_ENTROPY_BLOCK  1
+
+
+typedef psa_status_t (*psa_entropy_src_t)(uint32_t flags,
+                              size_t *estimate_bits,
+                              uint8_t *output,
+                              size_t output_size);
+
+#if defined(MBEDTLS_PSA_MAX_ENTROPY_SRCS)
+extern psa_entropy_src_t psa_entropy_sources[MBEDTLS_PSA_MAX_ENTROPY_SRCS];
```

Comment:
This shouldn't be an array of function pointers. The template generator has the list of function names, so it should code that calls all the functions.

Reply:
That allows to have the entropy accumulation code inside entropy.c instead of the wrapper. It also allows for a more compact binary. Do you want the entropy accumulation code to go in the driver wrapper instead?

Reply:
I'd like the driver wrapper to contain a simple loop calling the sources and storing their output into a buffer or feeding it to an auxiliary function. And the complicated logic should be separate from that, not in a generated file.

Reply:
I've totally rewrote the patch with that approach. It now adds a new type of source for the entropy collected from the PSA drivers.

The patch only shows the basic idea to be sure we're on the right track. If it looks good enough, I will rework it to clean it up and include some tests.